packages/wrangler/src/publish.ts

```diff
@@ -361,3 +358,29 @@ export default async function publish(props: Props): Promise<void> {
 function formatTime(duration: number) {
   return `(${(duration / 1000).toFixed(2)} sec)`;
 }
+
+async function getSubdomain(accountId: string): Promise<string> {
+  try {
+    const { subdomain } = await fetchResult(
+      `/accounts/${accountId}/workers/subdomain`
+    );
+    return subdomain;
+  } catch (e) {
+    const error = e as { code?: number };
+    if (typeof error === "object" && !!error && error.code === 10007) {
+      // 10007 error code: not found
+      // https://api.cloudflare.com/#worker-subdomain-get-subdomain
+      const errorMessage =
+        "Error: Set up a workers.dev subdomain before using workers_dev in wrangler.toml";
```

Comment:
Most folks won't use workers_dev in wrangler.toml at all since we infer it most of the time. Also I think the link isn't clickable?
```suggestion
        `Error: You need register a workers.dev subdomain before publishing to the Cloudflare Workers platform.\nYou can do so at https://dash.cloudflare.com/${accountId}/workers/onboarding `
```

Reply:
I'll alter the wording then, you can actually use the platform without registering a subdomain -- just can't publish to workers.dev, it has to be routes

Reply:
should the link...be clickable?

Reply:
it's clickable in my terminal but that might just be my terminal